cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/EvictMetaDataCacheCommand.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.segmentstore.storage.StorageCommand;
+import io.pravega.cli.admin.utils.AdminSegmentHelper;
+import io.pravega.shared.protocol.netty.PravegaNodeUri;
+import io.pravega.shared.protocol.netty.WireCommands;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+public class EvictMetaDataCacheCommand extends StorageCommand {
```

Comment:
Missing javadoc for this command. If this refers exclusively to SLTS metadata, change the name of the command, as otherwise is not clear to what metadata is this referring to.